src/unstructured_client/utils/_decorators.py

```diff
@@ -1,15 +1,28 @@
 from __future__ import annotations
 
 import functools
-from typing import cast, Callable, Optional
+from typing import cast, Callable, TYPE_CHECKING, Optional
 from typing_extensions import ParamSpec
 from urllib.parse import urlparse, urlunparse, ParseResult
+import warnings
+
+from unstructured_client.models import errors, operations
+
+if TYPE_CHECKING:
+    from unstructured_client.general import General
 
 
 _P = ParamSpec("_P")
 
 
 def clean_server_url(func: Callable[_P, None]) -> Callable[_P, None]:
+    """A decorator for fixing common types of malformed 'server_url' arguments.
+
+    This decorator addresses the common problem of users omitting or using the wrong url scheme and/or
+    adding the '/general/v0/general' path to the 'server_url'.
+    The decorator should be manually applied to the __init__ method of UnstructuredClient after merging
+    a PR from Speakeasy.
```

Comment:
Make this one paragraph or two, but put a blank line between paragraphs. I think one paragraph is plenty here.

Look for a word-wrap setting/keystroke in your editor to speed up reformatting between the margins.